specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2020-03-01/examples/Caches_Delete.json

```diff
@@ -6,14 +6,8 @@
     "api-version": "2020-03-01"
   },
   "responses": {
-    "200": {
-      "body": ""
-    },
-    "202": {
-      "body": ""
-    },
-    "204": {
-      "body": ""
-    }
+    "200": {},
```

Comment:
When does a 200 OK come back for this operation? I thought these were the only returns:

202 Accepted for an existing cache (+async op URL in header).
204 No Content for a non-existent cache.

Reply:
Similar question for Caches_Flush, Caches_Start, Caches_Stop, StorageTargets_Delete. However, for some, maybe 204 Not Accepted is the unnecessary return code?

These are comments in the context of examples. But really these need to be reviewed in the API context.

Reply:
UPDATE: The 200 OK codes might be because Azure API tools require them. Please confirm and, if so, disregard/resolve these comments.